sched/sched/sched_timerexpiration.c

```diff
@@ -238,7 +238,26 @@ static uint32_t nxsched_cpu_scheduler(int cpu, uint32_t ticks,
     {
```

Comment:
why not remove all KEEP_ALIVE_HACK stuff, and return zero directly?

Reply:
Maybe submit new patch to do this much more better? I prefer to do one thing in patch.

Reply:
It's fine to fix the problem step by step, but I would like we don't add SCHED_RR_IDLETASK to keep the buggy behavior. If you and all reviewer think this is problem, why still keep the old behavior? It's complicated the code and frustrate other people when he select the wrong option.